### PR TITLE
[FEATURE] Add compatibility to elasticsearch 5

### PR DESCRIPTION
**Pull request by psychomieze**

### CODE REVIEW

File: .env-example [hunk 1]
```diff
@@ -7,4 +7,4 @@ elastica.transport=
 elastica.persistent=
 elastica.timeout=
 elastica.username=
-elastica=password=
```
Comment (NeoBlack) [inline .env-example]: haha, nice catch :)

File: Tests/Fixtures/AlternativeConfigurationWithLanguages/footest/DocumentTypes/tweets.yaml [hunk 2]
```diff
@@ -1,2 +1,2 @@
 name:
-  type: integer
```
Comment (NeoBlack) [inline Tests/Fixtures/AlternativeConfigurationWithLanguages/footest/DocumentTypes/tweets.yaml]: sure? before: integer after: text?

Comment (psychomieze) [inline Tests/Fixtures/AlternativeConfigurationWithLanguages/footest/DocumentTypes/tweets.yaml]: yes, I changed the fixtures // tests a bit to make more sense :)

Comment (NeoBlack) [inline Tests/Fixtures/AlternativeConfigurationWithLanguages/footest/DocumentTypes/tweets.yaml]: ok

**Pull request merged**